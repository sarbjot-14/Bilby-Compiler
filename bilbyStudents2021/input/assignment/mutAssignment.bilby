main{
	mut first := 1.23;
	print first $n;
	first := 20.5;
	print first $n;
	
	mut second := 1.23;
	print first $n;
	first := 30.5;
	print first $n;
	
	
}