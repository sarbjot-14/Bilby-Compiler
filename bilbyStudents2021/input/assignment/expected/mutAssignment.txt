1.230000
20.500000
20.500000
30.500000
